tests/NoPromise.agent.test.nut

```diff
@@ -29,13 +29,16 @@ class NoPromiseTestCase extends ImpTestCase {
     _path = null;
     _firebase = null;
     _luckyNum = null;
+    _myPromise = null;
 
     function setUp() {
         if (getroottable().Promise != null) {
+            _myPromise = Promise;
```

Comment:
I would also add a comment over here not to forget what's going on and why is that (From the library POV we are running w/o the Promise class, while the impUnit requires to for async tests).